staging/src/slime.io/slime/modules/meshregistry/publish.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
refer to 

https://github.com/slime-io/slime/blob/master/staging/src/slime.io/slime/modules/lazyload/publish.sh